pkg/operator2/operator.go

```diff
@@ -62,7 +62,10 @@ const (
 	kasServiceAndEndpointName = "kubernetes"
 	kasServiceFullName        = kasServiceAndEndpointName + "." + corev1.NamespaceDefault + ".svc"
 
-	rootCAFile = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	rootCAFile            = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	operatorTrustedCAFile = "/var/run/configmaps/trusted-ca-bundle/ca-bundle.crt"
+
+	systemTrustStoreDirPath = "/etc/pki/ca-trust/extracted/pem"
```

Comment:
Remove this and use the same path as all of the system config.

Reply:
I believe this is only for the operand where we do actually want to use this patn

Reply:
Hmm okay, add a comment about that.